.github/workflows/nightly.yml

```diff
@@ -0,0 +1,149 @@
+name: Nightly build
+
+on:  
+  workflow_dispatch:
+  schedule:
+    - cron: "15 5 * * *" # 0515 UTC every day
```

Comment:
the readme says 5:30 utc, which are we picking? my vote is for 30 for a bit more breathing room

Reply:
Yes, that makes sense. Changed it here.